python/src/open_discourse/definitions/pattern.py

```diff
@@ -2,30 +2,35 @@
 
 FACTIONS = {
     "Bündnis 90/Die Grünen": regex.compile(
-        "(?:BÜNDNIS\s*(?:90)?/?(?:\s*D[1I]E)?|Bündnis\s*90/(?:\s*D[1I]E)?)?\s*[GC]R[UÜ].?\s*[ÑN]EN?(?:/Bündnis 90)?|Bündnis 90/Die Grünen"
+        r"(?:BÜNDNIS\s*(?:90)?/?(?:\s*D[1I]E)?|Bündnis\s*90/(?:\s*D[1I]E)?)?\s*[GC]R[UÜ].?\s*[ÑN]EN?(?:/Bündnis 90)?"
```

Comment:
oh wow I didn't noticed that early but all regex strings were wrong because we did not use raw regex strings? Wow that is a huge mistake. And this will lead to the same data as in the main script without the `r`?  That I cannot really believe...